stdlib/src/python/python_object.mojo

```diff
@@ -1325,6 +1325,27 @@ struct PythonObject(
         """
         return self._call_zero_arg_method("__invert__")
 
+    fn __contains__(self, rhs: PythonObject) raises -> Bool:
+        """Contains dunder.
+
+        Calls the underlying object's `__contains__` method.
+
+        Args:
+            rhs: Right hand value.
+
+        Returns:
+            True if rhs is in self.
+        """
+        # TODO: replace/optimize with c-python function.
```

Comment:
**Suggestion** Do you mind filing some follow-up GitHub issues in case you (or others) are interesting in tackling these.

Reply:
Yep, good idea !
will do it now and resolve it with the link here
(ping to @JoeLoser)